lms/templates/footer-edx-new.html

```diff
@@ -94,6 +94,22 @@ <h2 class="footer-follow-title">${_("Follow Us")}</h2>
           <span class="copy">${_("Google+")}</span>
         </a>
       </div>
+
+      % if settings.FEATURES.get('ENABLE_FOOTER_MOBILE_APP_LINKS'):
+      <div class="footer-mobile-apps">
+        <h2 class="footer-nav-title">${_("Mobile Apps")}</h2>
+        <div class="mobile-app-wrapper">
+          <a href="#">
```

Comment:
We won't have the link until the app store approves it. If this could be updated with a config change that would be perfect.